composer.json

```diff
@@ -2,8 +2,18 @@
     "name": "yola/yophpcs",
     "description": "Yola.com PHP Code Standard",
     "minimum-stability": "dev",
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "https://github.com/devshep/codeception.git"
```

Comment:
Incorrect. The last tag that I see is 2.0.9 for both github and packagist. My PR was merged in after those were tagged. We either have to get their 2.0 branch as is (which is considered a dev branch by the project) or wait until either 2.0.10 or 2.1 is released.

I just changed the dependency locally and ran my tests. Tests failed due to my PR not being pulled down by composer.